de/translation.json

```diff
@@ -157,7 +155,7 @@
   "Character.Compare.ObtainFromVendor": "Bezogen von {{name}}",
   "Character.DIMSync.Description": "Diese Funktion erfordert DIM Sync, um zu funktionieren. Aktiviere DIM Sync in den Einstellungen.",
   "Character.DIMSync.Name": "DIM Sync",
-  "Character.DiffProfile": "🦘 You're viewing someone else's profile. You can look but if you try to push buttons, don't expect results!",
+  "Character.DiffProfile": "Du schaust dir das Profil von einem anderen Spieler an. Du kannst schauen, aber wenn du versucht, Knöpfe zu drücken, erwarte keine Ergebnisse!",
```

Comment:
Not wrong, but a slightly more casual phrasing might be "du kannst dich umschauen" instead of "du kannst schauen".

Additionally, there's a typo in "versuchst", the `s` is missing.